layer/filestore.go

```diff
@@ -328,10 +328,12 @@ func (fms *fileMetadataStore) getOrphan() ([]roLayer, error) {
 					contentBytes, err := ioutil.ReadFile(chainFile)
 					if err != nil {
 						logrus.WithError(err).WithField("digest", dgst).Error("cannot get cache ID")
+						continue
 					}
 					cacheID := strings.TrimSpace(string(contentBytes))
 					if cacheID == "" {
 						logrus.Errorf("invalid cache id value")
```

Comment:
Looks like this should be `Error` instead of `Errorf`. Looks like we're not consistent with how we write `cache id` (it's `cache ID` a couple of lines up), and ` value` feels a bit redundant 😅 ;

```suggestion
						logrus.Error("invalid cache ID")
```